core/client/app/components/gh-datetime-input.js

```diff
@@ -15,8 +15,9 @@ export default Component.extend(TextInputMixin, {
     inputName: null,
 
     didReceiveAttrs() {
-        let datetime = this.get('datetime') || moment();
-
+        let datetime = this.get('datetime') || moment.utc();
+        console.log(datetime);
```

Comment:
This is where the bug with the date display is occurring. Because `publishedAtOffset` returns a promise, the `datetime` value passed into this component is a promise which the latter `formatDate(datetime)` call doesn't handle properly.

It would be possible to handle the promise object here by replacing `didReceiveAttrs` with something like:

```javascript
didReceiveAttrs() {
    let datetime = this.get('datetime') || RSVP.resolve(moment.utc());

    if (!this.attrs.update) {
        throw new Error(`You must provide an \`update\` action to \`{{${this.templateName}}}\`.`);
    }

    datetime.then((date) => {
        this.set('datetime', formatDate(date));
    });
},
```